flycheck.el

```diff
@@ -9199,8 +9199,10 @@ See URL `http://mypy-lang.org/'."
   :error-patterns
   ((error line-start (file-name) ":" line ":" column ": error:" (message)
           line-end)
+   (error line-start (file-name) ":" line ": error:" (message) line-end)
```

Comment:
Instead of duplicating the patterns, why not make the `":" column` part optional with the `(opt ` or `(? ` rx patterns?

Reply:
Good point! I followed an example from somewhere else in the source code and used `optional`; not sure what the difference is between `opt`, `optional` and `?` is.

Reply:
By the `rx` docstring, `opt`  and `optional` are aliases, but `?` causes that part to be greedy regexp, but I'm not sure it makes a difference here.